[docs] CDCSDK: Update CDC documentation with missing sections (#21512)
yugabyte/yugabyte-db: docs/content/stable/explore/change-data-capture/cdc-get-started.md

}
```

## Colocated tables

YugabyteDB supports streaming of changes from [colocated tables](../../architecture/docdb-sharding/colocated-tables). The connector can be configured with regular configuration properties and deployed for streaming.

{{< note title="Note" >}}

> If a connector is already streaming a set of colocated tables from a database and if a new table is created in the same database, one cannot deploy a new connector for this newly created table.

Reviewer: ```suggestion
If a connector is already streaming a set of colocated tables from a database and if a new table is created in the same database, you can't deploy a new connector for the newly created table.
```
